pkg/virt-api/BUILD.bazel

```diff
@@ -12,6 +12,7 @@ go_library(
         "//pkg/monitoring/metrics/common/client:go_default_library",
         "//pkg/monitoring/metrics/virt-api:go_default_library",
         "//pkg/monitoring/profiler:go_default_library",
+        "//pkg/network/admitter:go_default_library",
```

Comment:
Commit message: "Introduce the ValidateSpec function type" it was changed to `SpecValidator`.

Reply:
Fixed